### DISCUSSION TITLE
Render Encrypted PDFs Without Requiring a User Password

### DISCUSSION THREAD

**Discussion by ZHIFA-CHIU** [wojtekmaj/react-pdf#1885] [Q&A]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-PDF version
- [X] I have checked if this bug is not already reported
- [X] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
- [ ] If I have a problem with PDF rendering, I checked if my PDF renders properly in [PDF.js demo](https://mozilla.github.io/pdf.js/web/viewer.html)

### Description

I have an encrypted PDF that doesn’t require users to enter a password to view it. It renders correctly in Chrome. However, when I try to render it using react-pdf, I encounter the error: PasswordException: No password given.

Is this behavior expected, or is there a way to bypass this issue?

### Steps to reproduce

null

### Expected behavior

The PDF is rendered

### Actual behavior

Received an error

### Environment

- **Browser (if applicable)**:
- **React-PDF version**:
- **React version**:
- **Bundler name and version (if applicable)**:

**Comment by wojtekmaj** [comment]
README -> Ctrl+F -> "Password". There's a function that you can provide in place of the built-in one to handle entering passwords. The question is, how much sense does it have, because then you'd need to store PDF passwords on the front end... But that's another story.
